server/src/lfortran-accessors.ts

```diff
@@ -281,7 +287,21 @@ export class LFortranCLIAccessor implements LFortranAccessor {
     const fnid: string = "showDocumentSymbols(...)";
     const start: number = performance.now();
 
-    const flags = ["--show-document-symbols", "--continue-compilation"];
+    // run `echo "$(find . -type d | sed 's/^/-I/')"` and pass it as an argument to lfortran using spawnSync
+    // Execute the command within a shell
+    const find_output = spawnSync("echo $(find $(pwd) -type d | sed 's/^/-I/')", { shell: true });
+
+    // generate .mod files for the module
+    const module_flags = ["-c", "--continue-compilation"];
+    const run_module = await this.runCompiler(settings, module_flags, text, "[]");
```

Comment:
Does this mean lfortran now runs twice every time for each key stroke?

Reply:
Yes, it does.